src/group_theory/perm/cycles.lean

```diff
@@ -354,11 +354,13 @@ cycle_factors_aux (univ.sort (≤)) f (λ _ _, (mem_sort _).2 (mem_univ _))
 
```

Comment:
Can the def above this one be made computable?

Reply:
Which def do you mean? I've removed the word `noncomputable` from this file.

Reply:
I must haev been confused, apologies